molecule/scenario_resources/Dockerfile.j2

```diff
@@ -46,13 +35,6 @@ apt-get -y install \
   &&\
 rm /sbin/modprobe &&\
 ln -s /bin/true /sbin/modprobe &&\
-rm -f /lib/systemd/system/multi-user.target.wants/* \
```

Comment:
I think you might actually fix CentOS 7.7 here with this change :) (Re-introducing it is on my TODO list.) Several of these removed files get recreated when packages get updated, like from C 7.6 -> 7.7, but not all of them.

Also, my concern is that we are making big changes to this file, which I assume came from a template, and we have no comments on why we differ from the template. Do we know why they disabled them in the 1st place?

Reply:
I don't know, where this came from, but on the other hand, i saw that systemd recognizes that it is ran in a container, and starts very few services, which i believe those lines tried to do in a less clever way.

Reply:
Actually, i think disabling the firewall setup fixed centos (and by that time all the others, too).

Reply:
Gotcha. That's cool that systemd does that now.

Could you please put a comment in the Dockerfile.j2 itself where the block used to be? So that if we rebase in the future or diff against molecule, we'll know why the block was removed.

Reply:
I wrote some lines. But i must say it feels a little bit strange to write an obituary for deleted code.
I tried not to start with `Here used to be...`. ;)